.dev/config.toml

```diff
@@ -1,5 +1,8 @@
 httpPort = 4001
 
+[flags]
```

Comment:
Should we call this `features` to distinguish from the more general concept of options=flags?